test/atom/decoration.test.js

```diff
@@ -128,15 +129,40 @@ describe('Decoration', function() {
       assert.isNull(editor.gutterWithName(gutterName));
     });
 
-    it('throws an error if `gutterName` prop is not supplied for gutter decorations', function() {
-      const app = (
-        <Decoration editor={editor} decorable={marker} type="gutter">
-          <div className="decoration-subtree">
-            This is a subtree
-          </div>
-        </Decoration>
-      );
-      assert.throws(() => mount(app), 'You are trying to decorate a gutter but did not supply gutterName prop.');
+    describe('throws an error', function() {
+      let errors;
+
+      // This consumes the error rather than printing it to console.
+      const onError = function(e) {
+        if (e.message === 'Uncaught Error: You are trying to decorate a gutter but did not supply gutterName prop.') {
+          errors.push(e.error);
+          e.preventDefault();
+        }
+      };
+
+      beforeEach(function() {
+        errors = [];
+        window.addEventListener('error', onError);
+      });
+
+      afterEach(function() {
+        errors = [];
+        window.removeEventListener('error', onError);
+      });
+
+      it('if `gutterName` prop is not supplied for gutter decorations', function() {
+        const app = (
+          <ErrorBoundary>
+            <Decoration editor={editor} decorable={marker} type="gutter">
+              <div className="decoration-subtree">
+                This is a subtree
+              </div>
+            </Decoration>
+          </ErrorBoundary>
+        );
+        mount(app);
+        assert(errors[0], 'You are trying to decorate a gutter but did not supply gutterName prop.');
```

Comment:
This `assert()` call is only asserting that `errors[0]` is truthy, and using the expected message as a failure message. Here's what you likely mean:

```suggestion
        assert.strictEqual(errors[0], 'You are trying to decorate a gutter but did not supply gutterName prop.');
```